conf/webknossos.latest.routes

```diff
@@ -169,7 +170,9 @@ GET           /statistics/webknossos
 GET           /statistics/users                                                 controllers.StatisticsController.users(interval: String, start: Option[Long], end: Option[Long], limit: Int)
 
 #Organizations
-GET           /organizations                                                    controllers.OrganizationController.listAllOrganizations
```

Comment:
@fm3 The onboarding process still contains the possibility to join an existing organization, which is why this route is still needed. Or is the front-end part out of date and it shouldn't be possible anymore to follow the "onboarding steps" to join an existing organization?

Reply:
I think the front-end route part is out of date. The idea was that listing all organizations could be considered a data leak, so joining existing organizations should be possible only via invite link, meaning that this list request should be obsolete.